src/videoserver/apps/projects/routes.py

```diff
@@ -1092,7 +1110,8 @@ def post(self, project_id):
                     'width': metadata.get('width'),
                     'height': metadata.get('height'),
                     'size': metadata.get('size'),
-                    'position': 'custom'
+                    'position': 'custom',
+                    'urlToImageS3': os.getenv('AWS_DOMAIN') + file_name_s3
```

Comment:
would call this href, imo client doesn't need to know it's stored on s3 and might be different storage eventually